outputs.tf

```
@@ -3,10 +3,10 @@
 //
 output "assessment_template_arn" {
   description = "Assessment template ARN"
-  value       = join("", aws_inspector_assessment_template.this.*.arn)
+  value       = aws_inspector_assessment_template.this.*.arn
 }
 
 output "assessment_target_arn" {
   description = "Assessment target ARN"
-  value       = join("", aws_inspector_assessment_target.this.*.arn)
+  value       = aws_inspector_assessment_target.this.*.arn
```

Same here, as this is no longer a list, also remove the .* list syntax

```suggestion
  value       = aws_inspector_assessment_target.this.arn
```